go-controller/pkg/config/config.go

```diff
@@ -226,6 +226,8 @@ type GatewayConfig struct {
 	VLANID uint `gcfg:"vlan-id"`
 	// NodeportEnable sets whether to provide Kubernetes NodePort service or not
 	NodeportEnable bool `gcfg:"nodeport"`
+	// DisableSNATMultipleGws sets whether to disable SNAT of egress traffic in namespaces annotated with routing-external-gws
+	DisableSNATMultipleGws bool `gfcg:"disable-snat-multiple-gws"`
```

Comment:
Wait, how did we end up with "multiple" rather than "external" in the name again? It should be "external" shouldn't it?

(Also, whatever the name ends up being, "GWs" would be better than "Gws")

Reply:
@trozet commented in a previous commit. Thing is this option will not only be used by routing-external-gws where the router is supposed to be external to pod network, but also for having pod CNFs acting as routers.
Thus the option will be used for both, and external was not very descriptive.
I'm still not super convinced on the name of this option, if you have other suggestion per the above let me know.

Reply:
renamed it to *GWs

Reply:
It could also be DisableECMPEgressSNAT I dont really care, @danwinship pick one you like

Reply:
I thought it originally had "multiple" in the name because the option only applied to namespaces that had multiple external gateways (not just one external gateway), but then that restriction was removed, so "multiple" no longer made sense. So I was confused why it came back.

I don't care what it's called as long as it vaguely makes sense. I haven't paid enough attention to this feature to know what does and doesn't make sense.